packages/mdc-menu/README.md

```diff
@@ -201,6 +205,7 @@ Method Signature | Description
 `open({focusIndex: ?number}) => void` | Opens the menu. Optionally accepts an object with a `focusIndex` parameter to indicate which list item should receive focus when the menu is opened. 
 `close(evt: ?Event)` | Closes the menu. Optionally accepts the event to check if the target is disabled before closing the menu.  
 `isOpen() => boolean` | Returns a boolean indicating whether the menu is open. 
+`setQuickOpen(quickOpen: boolean) => void` | Sets the menu to open and close without animation when the `open/close` methods are called. 
```

Comment:
Nit #1: "Whether the menu should open and close ..."
Nit #2: `open`/`close` (separate backticks around each word, leave the slash outside them)

Reply:
I changed it to "Sets whether the menu should open and close..." based on every other "set" method description in this readme starting with "Sets".